### ISSUE TITLE
OpenApi schema type issue: integer vs. int (validation problem)

### ISSUE THREAD

**Issue by emololftw** [contributte/apitte#183]
**Problem:**
In definition of parameters selected type int like:

```
* @RequestParameters({
*      @RequestParameter(name="userId", type="int", in="path", description="Primary user's ID")
* })
```
OpenApi takes parameter userId as `string` type and validation of integer not working for this case.

**Explanation:**

Because OpenApi scheme defines `int` as `integer`

![image](https://user-images.githubusercontent.com/60448217/166500259-8bca0df0-fefc-4a58-83b6-c1a462774485.png)
[https://swagger.io/specification/](url)

if in class in `src\apitte\EndpointParameter` we kept contant:

```
public const TYPE_INTEGER = 'int';
```

result of validation is

![image](https://user-images.githubusercontent.com/60448217/166502581-026ef1a5-c0ec-4fc5-8768-190223cf7f49.png)

but, when parameter change into:

```
public const TYPE_INTEGER = 'integer';
```

![image](https://user-images.githubusercontent.com/60448217/166502976-14e05a42-e2bb-4425-9934-aff36322434a.png)

Voala, validation for **int** start working!

**Comment by Domm98CZ** [comment]
Hello,

I have same issue, so i did PR.

https://github.com/contributte/apitte/pull/184

Have a nice day :)

**Comment by f3l1x** [comment]
Solved via #184
